src/webview/WebViewScreen.tsx

```diff
@@ -196,6 +203,7 @@ function WebViewScreen({ route, navigation }: Props) {
           setCanGoForward(navState.canGoForward)
           handleSetNavigationTitle(navState.url, navState.title, navState.loading)
         }}
+        mediaPlaybackRequiresUserAction={!disableMediaPlaybackRequiresUserActionUrls.includes(uri)}
```

Comment:
Maybe we should only check the origin?

In that case I'd recommend we name the config `disabledMediaPlaybackRequiresUserActionOrigins` or `originsForDisableMediaPlaybackRequiresUserAction`

Reply:
yes that's a good idea, i've updated